test/xstream.js

```diff
@@ -139,8 +139,8 @@ describe('Cyclic Router - XStream', () => {
         },
       }
 
-      const history = createServerHistory('/some/route')
-      const router = makeRouterDriver(history, switchPath)(xs.never(), XSAdapter)
+      const history = createMemoryHistory('/some/route')
+      const router = makeRouterDriver(history, switchPath)(xs.never())
```

Comment:
Changing this to 

    makeRouterDriver(history, switchPath)(xs.of('/some/route'))

makes the test pass, I'm really not understanding this quite well enough to know if that makes the test invalid or not though?